pkg/machines/helpers.es6

```diff
@@ -22,50 +22,106 @@ import VMS_CONFIG from './config.es6';
 
 const _ = cockpit.gettext;
 
-export function toGigaBytes(amount, currentUnit) {
-    let result;
-    switch (currentUnit) {
-        case 'B':
-            result = amount / 1024 / 1024 / 1024;
-            break;
-        case 'KiB':
-            result = amount / 1024 / 1024;
-            break;
-        default:
-            console.error(`toGigaBytes(): unknown unit: ${currentUnit}`);
-            result = amount / 1;
-    }
-
-    if (result < 1) {
-        result = result.toFixed(2);
+export function toReadableNumber(number) {
```

Comment:
Please consider using `cockpit.format_bytes` for this, http://cockpit-project.org/guide/latest/cockpit-util.html.  Maybe we can fix it if it is broken for you.

Reply:
I looked at the documentation and there is no option to convert between units (i.e. from MiB to GiB). Only from bytes. Please correct me if I am wrong.

Maybe we can make an issue and move my helper methods to cockpit or lib in the future?
- supports conversion between units 
- there is a method (`convertToBestUnit`) for converting to and picking the highest unit.

Reply:
Sounds good.